subprojects/packagefiles/tinyfsm/meson.build

```diff
@@ -0,0 +1,56 @@
+project('tinyfsm', 'cpp',
+  version: '0.3.3',
+  default_options: ['cpp_std=c++11'])
+
+tinyfsm_dep = declare_dependency(include_directories: include_directories('include'))
+
+# build examples as compile tests
```

Comment:
If these are `build_by_default: false` then they won't be built in CI. :D I wonder what the solution there is, hmm...

If they can be run successfully as a smoketest, then they could be added as `test()`s and then `ninja test` will build them I guess.

Reply:
They are interactive, so no. I guess a `build_examples` option (disabled by default) is the best option here...

Reply:
Added a `build_examples` option